src/components/sign-up/index.js

```diff
@@ -0,0 +1,145 @@
+      }
+    } catch (error) {
+      if (error?.response?.data?.error?.message) {
+        setErrorMessage(eerror?.response?.data?.error?.message)
+        setTimeout(() => {
+          setErrorMessage('')
+        }, 2000)
+      } else {
+        console.error('An unexpected error happened:', error)
+      }
+    }
+  }
+  return (
+    <Container maxW='lg' py={{ base: '12', md: '24' }} px={{ base: '0', sm: '8' }}>
+      <Stack spacing='8'>
+        <Stack spacing='6'>
+          <Stack spacing={{ base: '2', md: '3' }} textAlign='center'>
+            <Heading size={useBreakpointValue({ base: 'xs', md: 'sm' })}>{t('login.sign-up-header.title')}</Heading>
+            <HStack spacing='1' justify='center'>
+              <Text color='muted'>{t('login.sign-up-header.text')}</Text>
+              <Button variant='link' as={Link} href='/user/login' colorScheme='blue'>
+                {t('login.sign-up-header.button')}
+              </Button>
+            </HStack>
+          </Stack>
+        </Stack>
+        <Box
+          py={{ base: '0', sm: '8' }}
+          px={{ base: '4', sm: '10' }}
+          bg={useBreakpointValue({ base: 'transparent', sm: 'bg-surface' })}
+          boxShadow={{ base: 'none', sm: useColorModeValue('md', 'md-dark') }}
+          borderRadius={{ base: 'none', sm: 'xl' }}
+        >
+          <Stack spacing='6' as='form' onSubmit={handleSubmit(handleSubmitSignUp)}>
+            <Stack spacing='5'>
+              {error ? <Text color='red.500'>{error}</Text> : ''}
+              <FormControl>
+                <FormLabel htmlFor='name'>{t('login.name.title')}</FormLabel>
+                <Input
+                  name='name'
+                  id='name'
+                  type='text'
+                  {...register('name', { required: t('login.name.required') })}
+                />
+                <Text color='red.400'>{errors?.name?.message}</Text>
+                <FormLabel htmlFor='email'>{t('login.email.title')}</FormLabel>
+                <Input
+                  name='email'
+                  id='email'
+                  type='email'
+                  {...register('email', { required: t('login.email.required') })}
+                />
+                <Text color='red.400'>{errors?.email?.message}</Text>
+                <PasswordField register={register} errors={errors} />
+              </FormControl>
+              <Text fontSize='sm' whiteSpace='nowrap' color='muted'>
+                {t('login.password.warning')}
+              </Text>
+            </Stack>
+            <Stack spacing='6'>
+              <Input
+                as={Button}
+                variant='solid'
+                cursor='pointer'
+                borderStyle='solid'
+                colorScheme='blue'
+                color='white'
+                borderColor='white.700'
+                bg='blue.600'
+                type='submit'
+              >
+                {t('login.create-account')}
+              </Input>
```

Comment:
I think it's unnecessary to make the `Input` as **Button like** component, since `Button` has already `type='submit'` property which plays the same role. And it's better to have in term of  same design/colors with other primary buttons.

```suggestion
              <Button
                type='submit'
                colorScheme='blue'
              >
                {t('login.create-account')}
              </Button>
```